extensions/wikia/GoogleDocs/GoogleDocs.php

```diff
@@ -11,41 +11,44 @@
  *
  */
 
-if (!defined('MEDIAWIKI')){
-    echo ('THIS IS NOT VALID ENTRY POINT.'); exit (1);
+if ( !defined( 'MEDIAWIKI' ) ) {
+    echo 'THIS IS NOT VALID ENTRY POINT.';
+    exit (1);
 }
 
-$wgHooks['ParserFirstCallInit'][] = "wfGoogleDocs";
+$wgHooks['ParserFirstCallInit'][] = 'wfGoogleDocs';
 
-$wgExtensionCredits['parserhook'][] = array(
-	'name' => "GoogleDocs4MW",
-	'descriptionmsg' => "googledocs-desc",
+$wgExtensionCredits['parserhook'][] = [
+	'name' => 'GoogleDocs4MW',
+	'descriptionmsg' => 'googledocs-desc',
 	'author' => "[http://www.wikia.com/wiki/User:TOR Lucas 'TOR' Garczewski]",
 	'url' => 'https://github.com/Wikia/app/tree/dev/extensions/wikia/GoogleDocs'
-);
+];
 
 //i18n
 $wgExtensionMessagesFiles['GoogleDocs4MW'] = __DIR__ . '/GoogleDocs.i18n.php';
 
 function wfGoogleDocs( Parser $parser ) {
-	$parser->setHook( "googlespreadsheet", "renderGoogleSpreadsheet" );
+	$parser->setHook( 'googlespreadsheet', 'renderGoogleSpreadsheet' );
 	return true;
 }
 
 function renderGoogleSpreadsheet( $input, $argv ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "$argv". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)